spec/core/v2/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -596,15 +595,33 @@ function sendPacket(
     // increment the sequence. Thus there are monotonically increasing sequences for packet flow for a given clientId
     nextSequenceSend[sourceChannelId]=sequence+1
     
-    // log that a packet can be safely sent
-    // Event Emission 
-    emitLogEntry("sendPacket", {
-      sourceId: sourceChannelId, 
-      destId: channel.counterpartyChannelId, 
-      sequence: sequence,
-      packet: packet,
-      timeoutTimestamp: timeoutTimestamp, 
+    // Event Emission for send packet
+    emitEvents("send_packet", {
+      sourceChannel: sourceChannelId, 
+      destChannel: channel.counterpartyChannelId,
```

Comment:
hm, counterparty channel id is equal to source channel on recv no? why not use packet fields here?